### PR TITLE
14094 conv_sty_parse_json_android

### PR DESCRIPTION
**Pull request by dominicmarmionswrve**
https://swrvedev.jira.com/browse/SWRVE-14094

### CODE REVIEW

**Comment by dominicmarmionswrve** [comment]
@seaders @Sergio-Mira @adam-govan please

**Comment by Sergio-Mira** [comment]
🚀

File: SwrveSDKTest/src/main/assets/conversation_campaign.json [hunk 1]
```diff
@@ -88,7 +133,13 @@
                   "fg": {
                     "type": "color",
                     "value": "#9bf000"
-                  }
+                  },
+                  "font_file": "2617fb3c279e30dd7c180de8679a2e2d33cf3552",
+                  "font_family": "my_awesome_font",
+                  "text_size": "123",
+                  "padding": [1, 2, 3, 4],
+                  "line_space": "789",
```
Comment (Sergio-Mira) [inline SwrveSDKTest/src/main/assets/conversation_campaign.json]: I think this is a float

File: SwrveSDKTest/src/main/assets/conversation_campaign.json [hunk 2]
```diff
@@ -88,7 +133,13 @@
                   "fg": {
                     "type": "color",
                     "value": "#9bf000"
-                  }
+                  },
+                  "font_file": "2617fb3c279e30dd7c180de8679a2e2d33cf3552",
+                  "font_family": "my_awesome_font",
+                  "text_size": "123",
```
Comment (Sergio-Mira) [inline SwrveSDKTest/src/main/assets/conversation_campaign.json]: This is an integer

**Pull request merged**